audit/projects/k8s-conform/service-accounts/service-capi-openstack@k8s-conform.iam.gserviceaccount.com/description.json

```diff
@@ -1,8 +0,0 @@
-{
-  "displayName": "service-capi-openstack",
-  "email": "service-capi-openstack@k8s-conform.iam.gserviceaccount.com",
-  "name": "projects/k8s-conform/serviceAccounts/service-capi-openstack@k8s-conform.iam.gserviceaccount.com",
-  "oauth2ClientId": "115191210752954465501",
-  "projectId": "k8s-conform",
-  "uniqueId": "115191210752954465501"
-}
```

Comment:
This is expected, this was me https://github.com/kubernetes/k8s.io/pull/1800#discussion_r604584394